aredis/pubsub.py

```diff
@@ -249,7 +249,7 @@ def run_in_thread(self, daemon=False):
         for pattern, handler in iteritems(self.patterns):
             if handler is None:
                 raise PubSubError("Pattern: '%s' has no handler registered")
-        loop = asyncio.get_event_loop()
+        loop = self.connection.loop
```

Comment:
Not sure if I should guard here for a connection being `None`. While it does not seem logical to run the pubsub in a thread before subscribing to a channel this might be nice to guard against.